readme.md

```diff
@@ -49,6 +67,28 @@ $ john --help
     Tagged 2 devDependencies as gray
 ```
 
+#### Windows
+
+```
+> john --help
+
+  Make npm3's flat dependencies easier to find and sort
+
+  Usage
+    > john
+
+  Options
+    --clear     Clear all hidden dependencies. [Default: false]
+    --deps      Hide dependencies. [Default: false]
+    --dev-deps  Hide devDependencies. [Default: false]
```

Comment:
I'm a bit unclear on the last two options. Is the `default` supposed to be `true` for `deps` and `dev-deps`?